src/http/chain-registry/FaucetRegistry.test.ts

```diff
@@ -0,0 +1,180 @@
+import { HttpClient } from '../client/http'
+import { IFaucet, IHttpGETRequest, IHttpPOSTRequest, IRegisterFaucetRequest } from '../types'
+
+import { FaucetRegistry } from './FaucetRegistry'
+
+const mockHTTPClient = {
```

Comment:
same comment on mocking this using jest-create-mock-instance

Reply:
Okay, in this case you are right I can better use `const mockHTTPClient = createMockInstance(HttpClient)`